llvm/docs/LangRef.rst

```diff
@@ -5418,6 +5418,8 @@ X86:
   operand will get allocated only to RAX -- if two 32-bit operands are needed,
   you're better off splitting it yourself, before passing it to the asm
   statement.
+- ``jR``: An 8, 16, 32, or 64-bit integer EGPR when EGPR feature is on.
```

Comment:
```suggestion
- ``jR``: An 8, 16, 32, or 64-bit gpr32 if EGPR is enabled.
```

The old statement is misleading b/c EGPR refers to R16-R31 only.

Reply:
[fb6ba87](https://github.com/llvm/llvm-project/pull/92338/commits/fb6ba87b4f678df51d9e9e0807401f883bdbaced)